DEVICES.md

```diff
@@ -74,21 +74,26 @@ In [brackets] is given the class name of device.
-|   | ACTUAL   | value.temperature             | °C   | number         | -  |     |      | ``/temperature(\..*)?$/``                  |
-|   | HUMIDITY | value.humidity                | %    | number         | -  |     |      | ``/humidity(\..*)?$/``                     |
-|   | BOOST    | switch.boost                  |      | boolean/number | W  |     |      | ``/^switch\.boost(\..*)?$/``               |
-|   | SWING    | level.mode.swing              |      | number         | W  |     |      | ``/swing$/``                               |
-|   | SWING    | switch.mode.swing             |      | boolean        | W  |     |      | ``/swing$/``                               |
-|   | UNREACH  | indicator.maintenance.unreach |      | boolean        |    | X   |      | ``/^indicator(\.maintenance)?\.unreach$/`` |
-|   | MAINTAIN | indicator.maintenance         |      | boolean        |    | X   |      | ``/^indicator\.maintenance$/``             |
-|   | ERROR    | indicator.error               |      |                |    | X   |      | ``/^indicator\.error$/``                   |
+| R | Name           | Role                          | Unit | Type           | Wr | Ind | Mult | Regex                                      |
+|---|----------------|-------------------------------|------|----------------|----|-----|------|--------------------------------------------|
+| * | SET            | level.temperature             | °C   | number         | W  |     |      | ``/temperature(\..*)?$/``                  |
+| * | MODE           | level.mode.airconditioner     |      | number         | W  |     |      | ``/airconditioner$/``                      |
+|   | SPEED          | level.mode.fan                |      | number         | W  |     |      | ``/(speed｜mode)\.fan$/``                   |
+|   | POWER          | switch.power                  |      | boolean/number | W  |     |      | ``/^switch\.power$/``                      |
+|   | POWER          |                               |      | boolean        | W  |     |      | ``/^switch$/``                             |
+|   | ACTUAL         | value.temperature             | °C   | number         | -  |     |      | ``/temperature(\..*)?$/``                  |
+|   | HUMIDITY       | value.humidity                | %    | number         | -  |     |      | ``/humidity(\..*)?$/``                     |
+|   | BOOST          | switch.boost                  |      | boolean/number | W  |     |      | ``/^switch\.boost(\..*)?$/``               |
+|   | SWING          | level.mode.swing              |      | number         | W  |     |      | ``/swing$/``                               |
+|   | SWING          | switch.mode.swing             |      | boolean        | W  |     |      | ``/swing$/``                               |
+|   | ELECTRIC_POWER | value.power                   | W    | number         | -  |     |      | ``/^value\.power$/``                       |
```

Comment:
Maybe we should adjust the script that produces the devices.md to separate those shared patterns? So they do not show up on every device. (Even more for unreach and so on)... 
Or is, there, perhaps, an easy way to hide the lines by default?

Reply:
Can you create an issue? But in general it must not all shared be present .. sooo good question whats best